chpl/chpl-resources/src/main/resources/environment.properties

```diff
@@ -23,6 +23,7 @@ keyLocation=SECRET
 uploadErrorEmailRecipients=SECRET
 #format MM/DD/YYYY
 cures.ruleEffectiveDate=07/01/2019
```

Comment:
I know it's not your change, but shouldn't we default this date to "way in the future"?

Reply:
I'll change it - probably just a typo.  I'll make it 07/01/2020